wp-content/themes/bh/views/donate/thank-you.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
This file is meant to be shown inside the iframe (redirected from iCount after successful transaction). Therefore I had to "strip" the header/footer (we're staying inside the template all along).